Production/python/get_mcm.py

```diff
@@ -23,6 +24,46 @@ class col:
     bold = '\033[1m'
     uline = '\033[4m'
 
+class getpyline:
```

Comment:
a bit odd to have a class called "getpyline", I might just call it "pyline"

Reply:
Done